bin/snapshots

```diff
@@ -17,12 +18,13 @@ set -e
 
 # process arguments
 
-declare PLAY_BRANCH="master"
```

Comment:
It also has some of my local configuration that will need to be removed before merging.